docs/db-cloud-shared/clients/vector-integration.md

```diff
@@ -0,0 +1,14 @@
+
+GreptimeDB can be used as a Vector Sink component to receive metrics. To integrate Vector with GreptimeDB, use the following configuration:
+
+```toml
+[sinks.my_sink_id]
+inputs = ["my-source-or-transform-id"]
+type = "greptimedb"
+endpoint = "<host>:4001"
```

Comment:
what about using <port> instead of the default port number `4001`?

Reply:
No, because the GreptimeCloud console does not have a `<port>` variable, which makes copy & paste inconvenient.

Custom port has added [here](https://github.com/GreptimeTeam/docs/pull/535/files#diff-9a202ab659a4010537bddec1a50cf80103dad0579fef344159c2e71780516dceR9) for GreptimeDB.